packages/fluent-theme/src/fluent/styles/DefaultButton.styles.ts

```diff
@@ -1,21 +1,66 @@
-import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
 import { fluentBorderRadius } from './styleConstants';
-import { NeutralColors } from '../FluentColors';
+import { getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
+import FluentTheme from '../FluentTheme';
+import { NeutralColors, CommunicationColors } from '../FluentColors';
 
-export const DefaultButtonStyles = (props: IButtonProps) => {
-  const { primary, theme } = props;
-  const { palette, semanticColors } = theme!;
+// TODO: "any" is used here to get around "is using xxx but cannot be named" TS error. Should be able to remove
+//        this 'any' once we upgrade to TS3.1+
+// tslint:disable-next-line:no-any
+export const DefaultButtonStyles: any = {
+  root: {
+    borderRadius: fluentBorderRadius,
+    backgroundColor: NeutralColors.white,
+    border: `1px solid ${NeutralColors.gray110}`,
+    ...getFocusStyle(FluentTheme, 1)
+  },
+  rootHovered: {
+    selectors: {
+      '.ms-Button--primary': {
+        backgroundColor: CommunicationColors.shade10
+      }
+    }
+  },
+  rootPressed: {
+    backgroundColor: NeutralColors.gray30
+  },
+  rootChecked: {
+    backgroundColor: NeutralColors.gray30
+  },
+  rootDisabled: {
+    backgroundColor: NeutralColors.gray20,
+    borderColor: NeutralColors.gray20
+  },
+
+  splitButtonMenuButton: {
+    background: 'transparent',
+    borderTopRightRadius: fluentBorderRadius,
+    borderBottomRightRadius: fluentBorderRadius,
+    border: `1px solid ${NeutralColors.gray110}`,
+    borderLeft: 'none'
+  },
 
-  return {
-    root: {
-      borderRadius: fluentBorderRadius,
-      backgroundColor: primary ? semanticColors.primaryButtonBackground : palette.white,
-      border: primary ? '' : `1px solid ${NeutralColors.gray110}`,
-      color: primary ? semanticColors.buttonText : palette.white
-    },
-    rootHovered: {
-      backgroundColor: primary ? palette.themeDarkAlt : NeutralColors.gray20,
-      border: primary ? '' : `1px solid ${NeutralColors.gray110}`
+  splitButtonContainer: {
+    selectors: {
```

Comment:
Oh man. It's really unfortunate that we need to use these selectors, but I understand how complicated `Button` is today. That said, I wonder if these are "styling knobs" that we would want to expose later as part of a more robust theming contract?